test/configCases/css/escaped-names/style.module.css

```
@@ -0,0 +1,49 @@
+.escaped-url-bare { background: \url(./img.png) }
+.plain-url { background: url(./img.png) }
+
+/* An escaped dash spells a custom property too, so the declaration and its
+   use have to scope to the same name or the variable comes out undefined. */
+.escaped-custom-property {
+	\2d\2d one: red;
+	color: var(--one);
+}
+.escaped-custom-property-use {
+	--two: blue;
+	color: var(\2d\2d two);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)